spec/unit/views/components/status_tag_spec.rb

```diff
@@ -179,8 +160,8 @@ def status_tag(*args)
       end
     end
 
-    context "when status is 'So useless', type is :ok, class is 'woot awesome' and id is 'useless'" do
-      subject { status_tag('So useless', :ok, class: 'woot awesome', id: 'useless') }
+    context "when status is 'So useless', class is 'woot awesome' and id is 'useless'" do
+      subject { status_tag('So useless', class: 'woot awesome', id: 'useless') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.